ios/sound-send/sound-send/ViewController.swift

```diff
@@ -9,17 +9,98 @@
 import UIKit
 
 class ViewController: UIViewController {
-
+    //the base url, change it when we have one i guess?
+    let baseurl = "http://www.soundsend.com"
```

Comment:
We've just been putting them where Arnav has put his in his code. But maybe we could declare it in a class function so it'll be available throughout the project